src/test/java/uk/gov/hmcts/reform/bulkscan/orchestrator/tasks/CleanupEnvelopesDlqTaskTest.java

```diff
@@ -120,4 +125,26 @@ public void should_not_process_messages_when_connection_exception_is_thrown() {
         assertThat(exception).isNull();
     }
 
+    @Test
+    public void should_log_messageId_for_invalid_messages() throws Exception {
```

Comment:
As it just verifies log output, this test is optional.

Reply:
I've deleted the test.